packages/babel-plugin-transform-function-name/test/fixtures/issues/7199/input.js

```diff
@@ -0,0 +1,8 @@
+const x = {
+  [null]: function () {},
+  [/regex/gi]: function () {},
+  [`y`]: function () {},
+  [`abc${y}def`]: function () {},
```

Comment:
This could produce duplicate names if there is another template string with for example `abc${other}def`, but not sure if this is critical or will produce problems.

Reply:
Since you can't reference to those functions using their name anyway (the name is only used for debuging), it shouldn't create problems.